pyproject.toml

```diff
@@ -56,11 +56,44 @@ packages = [
 [project.urls]
 Homepage = "https://github.com/pdudaemon/pdudaemon.git"
 
-[project.entry-points.console_scripts]
+[project.scripts]
 pdudaemon = "pdudaemon:main"
```

Comment:
This is what the docs say on this
![image](https://github.com/pdudaemon/pdudaemon/assets/26027314/c74baf57-45e9-449c-9183-e2a3f1167da5)


https://setuptools.pypa.io/en/latest/userguide/entry_point.html

Perhaps if a lower version of setuptools is expected this should be reflected in the `build-system.requires` and then the config option changed accordingly.